test/install/load.sql

```
@@ -0,0 +1,193 @@
+DO $$
+BEGIN
+  IF current_setting('test_factory.test_load_mode') NOT IN ('fresh', 'update', 'existing') THEN
+    RAISE EXCEPTION 'test_factory.test_load_mode must be fresh, update or existing, got %', current_setting('test_factory.test_load_mode');
+  END IF;
+END $$;
+
+-- test_role is test infrastructure, not part of what fresh/update/existing
+-- describe -- (re)create it idempotently in every mode. A real pg_upgrade
+-- target (existing mode) carries global objects like roles over, but don't
+-- assume that; a from-scratch "existing" target might not have it.
+SELECT NOT EXISTS (SELECT 1 FROM pg_roles WHERE rolname = :'test_role') AS need_role \gset
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)